cli/cmd/main.go

```diff
@@ -17,7 +17,7 @@ func main() {
 	app := cmd.App(version.Version)
 	if err := app.Execute(); err != nil {
 		if err != nil {
-			panic(err)
+			os.Exit(1)
```

Comment:
I kinda like the way palantir does this. https://github.com/palantir/bulldozer/blob/develop/main.go
Not for right now, but worth looking at